api/filesystem/write.go

```diff
@@ -7,6 +7,12 @@ import (
 	"github.com/pkg/errors"
 )
 
+// WriteToFile create a file in the filesystem storage
+func (service *Service) WriteToFile(path string, data []byte) error {
+	return WriteToFile(path, data)
```

Comment:
this should use our trusted root method

Reply:
this is just a wrapper round existing write method, do you mean the existing write should be changed?

Reply:
not sure :) I meant sthing like https://github.com/portainer/portainer/blob/dff74f0823fca89636b9f3686757cc3f381c732e/api/filesystem/filesystem.go#L79